### PR TITLE
update tu graz faculty (new members, add newly created scholar pages)

### PR DESCRIPTION
**Pull request by dgruss**
# Contributing to CSrankings

Thanks for contributing to CSrankings! Here are some guidelines to getting your pull request accepted.

0. Use a reasonable title that explains what the PR corresponds to (as in, not "Update csrankings-x.csv"). **IF YOU DO NOT DO THIS, YOUR COMMIT WILL BE SUMMARILY REJECTED.**

1. _Do not use Excel to edit any .csv files; Excel incorrectly tries to
convert some Google Scholar entries to formulas, corrupting the
database. Use a text editor like emacs or NotePad instead._

2. _Insert new faculty **in alphabetical order**, not at the end of `csrankings-[a-z].csv`._ **Do not modify `csrankings.csv`, which is auto-generated.** **DO NOT MODIFY `csrankings-[0-9].csv`, WHICH ARE BEING DEPRECATED. USE `csrankings-[a-z].csv` INSTEAD.**

3. _Read and check **all** the boxes below by filling them in with an X._

**Inclusion criteria**

- [X] Make sure that any faculty you add meet the inclusion
criteria. Eligible faculty include only full-time, tenure-track research
faculty members on a given campus who can *solely* advise PhD students in
Computer Science. Faculty not in a CS department or similar who can
advise PhD students in CS can be included regardless of their home
department. Faculty should also have a 75%+ time appointment (check
`old/industry.csv` for faculty who are now more than 25% in industry).

**Updating an affiliation or home page**

- [X] Update affiliations, home pages, and Google Scholar entries by modifying `csrankings-[a-z].csv`. For the Google Scholar entry, just use the alphanumeric identifier in the middle of the URL. If none is there, put NOSCHOLARPAGE.

**Adding one or more faculty members (including an entire department)**

- [X] If the department is not yet listed in CSrankings, the entire faculty needs to be added (not just one faculty member).

- [X] Enter each faculty member's [DBLP](http://dblp.org) name, home page, and Google Scholar entry (just the alphanumeric identifier, not the whole URL) by modifying `csrankings-[a-z].csv` (**the letters correspond to the first letter of the faculty members' names**; please **do not** use the numeric suffixes, which are being obsoleted); include disambiguation suffixes like 0001 as needed. If the faculty entry is currently ambiguous, please do not include them. Send mail to the DBLP maintainers (dblp@dagstuhl.de) with a few publications by a particular faculty member; also, open an issue so that when the DBLP database is updated, that faculty member's information can be added.

- [X] If DBLP has multiple entries for this person, all of them need to be listed. If an alias is not already present in `dblp-aliases.csv`, add it.

- [X] If the institution you are adding is not in the US,
update `country-info.csv`.

**(Advanced) Quick contribution via a shallow clone** 

A full clone of the CSrankings repository is almost 2GB.  To contribute a change without creating a full local clone of the CSrankings repo, you can do a _shallow clone_.  To do so, follow these steps:

1. Fork the CSrankings repo.  If you have an existing fork, but it is not up to date with the main repository, this technique may not work.  If necessary, delete and re-create your fork to get it up to date.  (Do **not** delete your existing fork if it has unmerged changes you want to preserve!)
2. Do a shallow clone of your fork: `git clone --depth 1 https://github.com/yourusername/CSrankings`.  This will only download the most recent commit, not the full git history.
3. Make your changes on a branch, push them to your clone, and create a pull request on GitHub as usual.

If you want to do another contribution and some time has passed, perform steps 1-3 again, creating a fresh fork and shallow clone.

### CODE REVIEW

File: csrankings-g.csv [hunk 1]
```diff
@@ -344,6 +344,7 @@ Gerhard A. Bauch,Hamburg University of Technology,https://www.tuhh.de/nt/persone
 Gerhard Bauch 0001,Hamburg University of Technology,https://www.tuhh.de/nt/personen/professoren/gerhard-bauch/biographie.html,NOSCHOLARPAGE
 Gerhard Brewka,University of Leipzig,http://www.informatik.uni-leipzig.de/~brewka,wAxbPL0AAAAJ
 Gerhard Fohler,TU Kaiserslautern,https://rts.eit.uni-kl.de/index.php?id=18&people_action=detail&people_id=3,NOSCHOLARPAGE
+Gerhard G. Thallinger,Graz University of Technology,https://genome.tugraz.at/people.shtml,aMG6ceMAAAAJ
```
Comment (emeryberger) [inline csrankings-g.csv:347]: Not obviously in CS; please clarify how they meet the inclusion criteria.

File: csrankings-g.csv [hunk 2]
```diff
@@ -355,6 +356,7 @@ Gerhard P. Hancke Jr.,City University of Hong Kong,http://www.cityu.edu.hk/stfpr
 Gerhard Petrus Hancke,City University of Hong Kong,http://www.cityu.edu.hk/stfprofile/gphancke.htm,C2iR3xUAAAAJ
 Gerhard Sagerer,Bielefeld University,http://aiweb.techfak.uni-bielefeld.de/user/sagerer,HDmPKX4AAAAJ
 Gerhard Schwabe,University of Zurich,http://www.ifi.uzh.ch/imrg/people/schwabe.html,_a7cvK0AAAAJ
+Gerhard Sommer,Graz University of Technology,https://www.biomech.tugraz.at/index.php/people/gerhard_sommer,gD2-pFUAAAAJ
```
Comment (emeryberger) [inline csrankings-g.csv:359]: Same inclusion question here.

File: csrankings-k.csv [hunk 3]
```diff
@@ -477,6 +477,7 @@ Kerry Taylor,Australian National University,https://cs.anu.edu.au/people/kerry-t
 Kerstin Bach,NTNU,https://www.ntnu.edu/employees/kerstin.bach,DJzQ1J8AAAAJ
 Kerstin Eder,University of Bristol,http://www.bris.ac.uk/engineering/people/kerstin-i-eder/index.html,SyDaZIoAAAAJ
 Kerstin I. Eder,University of Bristol,http://www.bris.ac.uk/engineering/people/kerstin-i-eder/index.html,SyDaZIoAAAAJ
+Kerstin Lenk,Graz University of Technology,https://www.tugraz.at/institute/ine/people/kerstin-lenk/,ZAX2F5gAAAAJ
```
Comment (emeryberger) [inline csrankings-k.csv:480]: Same inclusion eligibility question.

File: csrankings-s.csv [hunk 4]
```diff
@@ -1795,7 +1795,7 @@ Sujit Gujar,IIIT Hyderabad,https://www.iiit.ac.in/people/faculty/sujit.g,9yxKajs
 Sujit Kumar Chakrabarti,IIIT Bangalore,http://www.iiitb.ac.in/faculty_page.php?name=SujitKumarChakrabarti,NOSCHOLARPAGE
 Sujoy Ghose,IIT Kharagpur,http://www.myopencourses.com/profile/prof-sujoy-ghosh,swQD_vgAAAAJ
 Sujoy Sikdar,Binghamton University,https://www.binghamton.edu/computer-science/contact/profile.html?id=ssikdar,OmZaKTEAAAAJ
-Sujoy Sinha Roy,University of Birmingham,https://www.birmingham.ac.uk/staff/profiles/computer-science/roy-sinha-sujoy.aspx,58_wa_4AAAAJ
+Sujoy Sinha Roy,Graz University of Technology,https://www.iaik.tugraz.at/person/sujoy-sinha-roy/,58_wa_4AAAAJ
```
Comment (emeryberger) [inline csrankings-s.csv:1798]: This institute does not offer PhDs. Please address the inclusion eligibility question as well, since not clearly in CS.

**Review by emeryberger — CHANGES_REQUESTED** [review]

File: csrankings-s.csv (hunk 4 above)
Comment (dgruss) [reply csrankings-s.csv:1798]: Of course does Sujoy offer PhDs at this institute. I offer PhDs at this institute as well.

**Comment by dgruss** [comment]
All faculty members listed are members of the CSBME Faculty of TU Graz, see https://www.tugraz.at/fakultaeten/csbme/faculty/.
The check list says:
> Eligible faculty include only full-time, tenure-track research
faculty members on a given campus who can solely advise PhD students in
Computer Science.

As faculty members they can advise PhD students in our CS doctoral school and thus should fulfill the eligibility criteria.

File: csrankings-s.csv (hunk 4 above)
Comment (emeryberger) [reply csrankings-s.csv:1798]: The web page does not list a PhD offering, AFAICT?

File: csrankings-g.csv (hunk 1 above)
Comment (emeryberger) [reply csrankings-g.csv:347]: Not listed here: https://www.tugraz.at/fakultaeten/csbme/faculty/team/faculty/

**Comment by emeryberger** [comment]
Thallinger is not listed in https://www.tugraz.at/fakultaeten/csbme/faculty/team/faculty/. All others are fine.

**Comment by dgruss** [comment]
I'm not sure why he is not listed in the faculty members list, but as this is a manually maintained website, someone might have missed him.
Thallinger is listed in our official university system: https://online.tugraz.at/tug_online/pl/ui/$ctx/visitenkarte.show_vcard?pPersonenId=8BD2BA2FB087CC2C&pPersonenGruppe=3
and here in the institute listing: https://online.tugraz.at/tug_online/pl/ui/$ctx/webnav.navigate_to?corg=35306&cperson_nr=2421
which are pages generated from the university's database.
and as for the naming: "informatics" is a less common translation of the german word "informatik", which is usually translated as "computer science"

**Pull request merged**